evaluators/embedding/L1NormEvaluator/__init__.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+
+from jina.executors.evaluators.embedding import BaseEmbeddingEvaluator, expand_vector
+
+
+class L1NormEvaluator(BaseEmbeddingEvaluator):
+    """A :class:`L1NormEvaluator` evaluates the distance between actual and desired embeddings computing
+    the L1 Norm between them
+    """
+
+    @property
+    def metric(self):
+        return 'L1NormEvaluator'
+
+    def evaluate(self, actual: 'np.array', desired: 'np.array', *args, **kwargs) -> float:
```

Comment:
this is `static method`

Reply:
or outside the class

Reply:
Added it similar to other `text` and `rank` evaluators